model_loader.py

```
@@ -4,10 +4,26 @@
+        super().__init__(**kwargs)
+        self.target_dtype = dtype
+
+    def call(self, inputs):
+        return tf.cast(inputs, self.target_dtype)
+
+    def get_config(self):
+        config = super().get_config()
+        config.update({"dtype": self.target_dtype})
+        return config
+
+def load_model(model_path: str = "./models/team12-correction.h5"):
```

Ruff (0.11.9)

Missing return type annotation for public function `load_model`

(ANN201)